users/tests/tests.py

```diff
@@ -4,39 +4,86 @@
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 
-from users.forms import RegistrationForm
+from users.forms import LoginForm, RegistrationForm
 
 
 class UserFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = User
 
     username = 'default_username'
-    password = 'pass1234'
+    password = factory.PostGenerationMethodCall('set_password', 'pass1234')
 
 
 class RegistrationFormUnitTests(TestCase):
-    def test_form_is_valid(self):
+    def test_register_form_is_valid(self):
         form_data = {'username': 'testUser', 'password1': 'pass1234', 'password2': 'pass1234'}
         form = RegistrationForm(data=form_data)
         self.assertTrue(form.is_valid())
 
-    def test_form_is_invalid(self):
+    def test_register_form_is_invalid(self):
         form_data = {'username': 'testUser', 'password1': 'pass1234', 'password2': 'pass123'}
         form = RegistrationForm(data=form_data)
         self.assertFalse(form.is_valid())
 
 
+class LoginFormUnitTests(TestCase):
+    def test_login_form_is_valid(self):
+        UserFactory(username='existing_user')
+        form_data = {'username': 'existing_user', 'password': 'pass1234'}
+        form = LoginForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_login_form_is_invalid(self):
+        form_data = {'username': 'non_existing_user', 'password': 'pass1234'}
+        form = LoginForm(data=form_data)
+        self.assertFalse(form.is_valid())
+
+
 class UserRegistrationIntegrationTests(TestCase):
     def test_redirection_on_register_successful(self):
         response = self.client.post(reverse('users:register'), {'username': 'user_1', 'password1': 'pass1234', 'password2': 'pass1234'})
         self.assertEqual(response.status_code, 302)
 
     def test_register_successful(self):
         response = self.client.post(reverse('users:register'), {'username': 'user_1', 'password1': 'pass1234', 'password2': 'pass1234'}, follow=True)
-        self.assertIn('You successfully created a new account!', response.content)
+        self.assertIn('You successfully created your account, now you can sing in!', response.content)
```

Comment:
You should import `UserRegistration` and use `success_message`. In case it changes, you want have to change this test.